package/fa_piaware_config.tcl

```diff
@@ -985,45 +1009,45 @@ namespace eval ::fa_piaware_config {
-			{"wired-network"         -type boolean -default yes}
-			{"wired-type"            -type network_type -default "dhcp"}
-			"wired-address"
-			"wired-netmask"
-			"wired-broadcast"
-			"wired-gateway"
-			{"wired-nameservers"     -default {8.8.8.8 8.8.4.4}}
-
-			{"wireless-network"      -type boolean -default no}
-			"wireless-ssid"
-			{"wireless-password"     -protect 1}
-			{"wireless-type"         -type network_type -default "dhcp"}
-			"wireless-address"
-			"wireless-netmask"
-			"wireless-broadcast"
-			"wireless-gateway"
-			{"wireless-nameservers"  -default {8.8.8.8 8.8.4.4}}
-			{"wireless-country"      -type country -default "00"}
-
-			{"allow-dhcp-duid"       -type boolean -default yes}
-
-			"http-proxy-host"
-			"http-proxy-port"
-			"http-proxy-user"
-			{"http-proxy-password"   -protect 1}
+			{"wired-network"         -type boolean -default yes -sdonly 1 -network 1}
+			{"wired-type"            -type network_type -default "dhcp" -sdonly 1 -network 1}
+			{"wired-address"         -sdonly 1 -network 1}
+			{"wired-netmask"         -sdonly 1 -network 1}
+			{"wired-broadcast"       -sdonly 1 -network 1}
+			{"wired-gateway"         -sdonly 1 -network 1}
+			{"wired-nameservers"     -default {8.8.8.8 8.8.4.4} -sdonly 1 -network 1}
+
+			{"wireless-network"      -type boolean -default no -sdonly 1 -network 1}
+			{"wireless-ssid"         -sdonly 1 -network 1}
+			{"wireless-password"     -protect 1 -sdonly 1 -network 1}
+			{"wireless-type"         -type network_type -default "dhcp" -sdonly 1 -network 1}
+			{"wireless-address"      -sdonly 1 -network 1}
+			{"wireless-netmask"      -sdonly 1 -network 1}
+			{"wireless-broadcast"    -sdonly 1 -network 1}
+			{"wireless-gateway"      -sdonly 1 -network 1}
+			{"wireless-nameservers"  -default {8.8.8.8 8.8.4.4} -sdonly 1 -network 1}
+			{"wireless-country"      -type country -default "00" -sdonly 1 -network 1}
+
+			{"allow-dhcp-duid"       -type boolean -default yes -sdonly 1 -network 1}
+
+			{"http-proxy-host"       -network 1}
+			{"http-proxy-port"	 -network 1}
+			{"http-proxy-user"       -network 1}
+			{"http-proxy-password"   -protect 1 -network 1}
 
 			{"adept-serverhosts"     -default {piaware.flightaware.com piaware.flightaware.com {70.42.6.197 70.42.6.198 70.42.6.191 70.42.6.225 70.42.6.224 70.42.6.156}}}
 			{"adept-serverport"      -type integer -default 1200}
 
-			{"rfkill"                -type boolean -default no}
+			{"rfkill"                -type boolean -default no -sdonly 1}
 			{"receiver-type"         -type receiver -default rtlsdr}
-			{"rtlsdr-device-index"   -default 0}
-			{"rtlsdr-ppm"            -type integer -default 0}
-			{"rtlsdr-gain"           -type gain -default max}
+			{"rtlsdr-device-index"   -default 0 -sdonly 1}
+			{"rtlsdr-ppm"            -type integer -default 0 -sdonly 1}
+			{"rtlsdr-gain"           -type gain -default max -sdonly 1}
 			{"beast-baudrate"        -type integer}
```

Comment:
beast-baudrate is also sdcard-only

Reply:
also all the uat-sdr-* settings

Reply:
also network-config-style, though I highly doubt that anyone is gonna try to set that one..

Reply:
Thanks, updated.